webpack/assets/javascripts/react_app/components/HostDetails/Tabs/Details/index.js

```diff
@@ -18,7 +19,7 @@ const DetailsTab = ({ response }) => (
     </Grid>
     <Grid className="details-cards">
       <GridItem span={3} rowSpan={3}>
-        <Properties hostData={response} />
+        <Properties hostData={response} isLoading={status === STATUS.PENDING} />
```

Comment:
can the `isLoading` be computed by the `EmptyLine` component by the given API key?
```js
// EmptyLine component
const isLoading = useSelector(selectAPIStatus(API_KEY)) === STATUS.PENDING
```

Reply:
I think the component should get `isLoading` as some components have different definitions to loading and some might not "know" what is the API key